Source/Core/Common/StringUtil.h

```diff
@@ -147,8 +148,22 @@ std::string ValueToString(T value)
 // Generates an hexdump-like representation of a binary data blob.
 std::string HexDump(const u8* data, size_t size);
 
-// TODO: kill this
-bool AsciiToHex(const std::string& _szValue, u32& result);
+namespace Common
+{
+template <typename T, typename std::enable_if_t<std::is_integral_v<T>>* = nullptr>
+std::from_chars_result FromChars(std::string_view sv, T& value, int base = 10)
+{
+  const char *const first = sv.data(), *const last = first + sv.size();
```

Comment:
IMO it'd be better to have these on two separate lines to avoid the somewhat confusing rules about what is/isn't a pointer. Same with the other overload.

Reply:
If I weren't trying to get a PR approved, I would call this a skill issue.

Reply:
I thought Dolphin's style guide actually prohibited this already, but it doesn't seem to (and I'm not sure which project it was that did, as a few others I checked also don't mention it). I have a (hopefully correct) understanding of how it behaves, but it still seems like something that'd be easy to get wrong in other cases and it's not making it a 3-line function instead of 2 harms readability.

Reply:
The type safety guarantees in C++ prevent you from getting very far if you were to get it wrong, and I find it visually pleasing to use more horizontal space (within the column limit) to save lines and extra typing.  Agree to disagree, I guess.